### PR TITLE
Add support for running Behat tests in containers

### PR DESCRIPTION
**Pull request by vidarl**
This PR basically adds the following functionality:
 - sahi container
 - Possibility to provide template for kickstart.ini
 - Possibility to patch SetupWizard so that "continue=true" works for welcome pages ( makes it possible to make setup wizard fully automated using kickstart )

### CODE REVIEW

**Comment by vidarl** [comment]
@andrerom : ready for review

File: dockerfiles/sahi/browser_types.xml-dist [hunk 1]
```diff
@@ -0,0 +1,924 @@
+
+
+
+
+<!DOCTYPE html>
```
Comment (andrerom) [inline dockerfiles/sahi/browser_types.xml-dist]: seems to be the wrong content in this file, or?

**Comment by joaoinacio** [comment]
Just a thought, slightly OT, but how about `phantomjs` in alternative to `firefox`, for headless execution?
(installation should be fairly pain-free, ref: https://sahipro.com/docs/using-sahi/sahi-headless-execution-with-phantomjs.html)

File: dockerfiles/sahi/browser_types.xml-dist (hunk 1 above)
Comment (vidarl) [inline dockerfiles/sahi/browser_types.xml-dist]: you seems to be right. I guess I have downloaded it from github using wget and forgot to download the RAW version.

Anyway, the file is not used, so I'll remove it

**Comment by andrerom** [comment]
+1

**Pull request merged**